src/sources/eventstoredb_metrics/types.rs

```diff
@@ -1,5 +1,7 @@
+use once_cell::sync::Lazy;
```

Comment:
Nit: We can use [std](https://doc.rust-lang.org/nightly/std/index.html)::[cell](https://doc.rust-lang.org/nightly/std/cell/index.html)::[LazyCell](https://doc.rust-lang.org/nightly/std/cell/struct.LazyCell.html#) instead.

Reply:
That's still nightly only, is it not?

Reply:
yep, the one stabilized is `OnceCell` https://doc.rust-lang.org/std/cell/struct.OnceCell.html

Reply:
oh, I see that std `LazyCell` is coming in 7 days with 1.80!

Reply:
Our existing pattern is to use `once_cell::sync::Lazy` so I'd rather stick with that and update all usages at once if we decide to do so.